src/advantest/smt8/origen/test_methods/Functional_test.java

```diff
@@ -109,13 +110,26 @@ public Functional_test overlaySubroutinePerSite(
     return this;
   }
 
+  //overload for full path pattern (from PGM_SOC_DATA) Boolean not used at the moment
+  public Functional_test overlaySubroutinePerSite(String subroutinePattern,
+          MultiSiteLong decData, int size, Boolean FullPathPattern ) {
+      Origen.overlaySubroutinePerSite(subroutinePattern,"NVM_DIN_PIN", decData, size);
```

Comment:
We can't have a reference to a project-specific pin in here (NVM_DIN_PIN), this will either need to be passed in as a arg, or handled in some other way

Reply:
see above